.buildkite/commands/configure-for-release.sh

```diff
@@ -1,13 +1,16 @@
 #!/bin/bash -eu
 
+if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
+  echo "This script must be 'source'd (instead of being called directly as an executable) to work properly"
+  exit 1
+fi
+
 # The Git command line client is not configured in Buildkite.
 # At the moment, steps that need Git access can configure it on deman using this script.
 # Later on, we should be able to configure it on the agent instead.
-
-curl -L https://api.github.com/meta | jq -r '.ssh_keys | .[]' | sed -e 's/^/github.com /' >> ~/.ssh/known_hosts
+add_host_to_ssh_known_hosts github.com
 git config --global user.email "mobile+wpmobilebot@automattic.com"
 git config --global user.name "Automattic Release Bot"
```

Comment:
Following up on @AliSoftware's suggestion from one of our private repos, I think if we moved these three line in `use-bot-for-git` would then allow us to call the command directly without having to wrap it in this script, right?

Reply:
I've updated the agents and the code here accordingly: 477184657d7cc0e541bb5f9e781a282b4af724fd (Buildkite test run [here](https://buildkite.com/automattic/wordpress-ios/builds/24295)).